### ISSUE TITLE
The latest versions (1.5 and 1.6) are not functioning properly

### ISSUE THREAD

**Issue by alentor** [DavidPhillipOster/ThumbHost3mf#13]
Hello,

I’ve been using this project for about six months, starting with version 1.2. I recently decided to upgrade to version 1.5 and then to 1.6, but it seems that the latest versions aren’t functioning properly. In contrast, version 1.2 works perfectly.

Here are the steps to reproduce the issue:

1. Download the 1.6 release version as a zip file.
2. Unzip the downloaded file.
3. Move the contents of the zip to the Application folder.
4. Open Finder and preview a .3mf file.

Expected result: The preview should work.
Actual result: It does not work.

Meanwhile, I can perform the same exact steps with version 1.2, and it functions correctly.

Could you please take a look at it?

Please See attached image for Hardware and OS version

<img width="265" alt="Screenshot 2024-09-27 at 10 00 36" src="https://github.com/user-attachments/assets/c39b4436-3958-4c5f-a33f-30313386d2ed">

**Comment by DavidPhillipOster** [comment]
I was not able to reproduce this. Note: you did not follow the installation directions on this repository

Download **and run** the ThumbHost3MF app

You have to place the app in Applications, and **run** it for its plugin to be registered.

I used a macOS Sonoma machine, inserted a SD card in FAT 32 format, and verified using TextEdit that the .gcode files on that SD card contained thumbnails, but the thumbnails were not shown in Finder. I ejected the SD card,  fetched 1.6 from github, followed the installation directions, and reinserted the SD card. This time the .gcode files had their thumbnail icons.

I'm going to close this issue now, but thank you for reporting it. I will reread the README to see if I can make the installation directions more clear.

If the problem recurs please re-open this issue with more details.

**Comment by alentor** [comment]
I did follow your instructions, just forgot to mention that step.

After copying the application (1.6) over to Applications, I have ran it. But not preview of 3mf was working, even after reboot.

When doing the same with 1.2 everything works, e.g. there is preview of 3mf files in Finder

**Comment by alentor** [comment]
Can't re-open the issue, could you please re-open it until it is solved?

Guess I'm not the only who is facing this issue, Waited until 1.6 is released before raising it up, have tried 1.5 and 1.6 with no success.. 

Currently using 1.2.

@DavidPhillipOster   @kaisamng

**Comment by DavidPhillipOster** [comment]
On Sequoia, on an M1 Mac without the plugin installed, version 1.6 install gives the following notification from the system, then works.  Is the problem Intel Sequoia only?  (I did verify that 1.6 has both arm64 and x86_64 architectures.)

<img width="388" alt="Sequoia Quicklook Notification" src="https://github.com/user-attachments/assets/be52dc4c-b391-4d3c-a366-8c5d0f4e70f2">


I need some help: I don't have an Intel Sequoia system to test with. I've not been able to make 1.6 fail. Can you give me a really detailed report of **exactly** what you did that failed to work?

@kaisamng @alentor

**Comment by alentor** [comment]
Thank you for your quick response!

Yes, I noticed that notification when I first installed version 1.6, but it didn’t appear during subsequent reinstallations.

I tried installing 1.6 using these steps:

1. Extracting the zip file.
2. Copying its contents to the Applications folder.
3. Running it from Applications.

Unfortunately, Quick Look is still not functioning.
I checked the settings, and it was enabled; I disabled and re-enabled it, but there's still no Quick Look for 3mf files..

<img width="469" alt="Screenshot 2024-09-28 at 9 23 45" src="https://github.com/user-attachments/assets/ba037362-eafc-481a-bb35-73936fda8b8b">

I have re-installed version 1.2, and it appears as 1.2 also stopped working now /cry

**Comment by alentor** [comment]
Resolved the issue! It turns out I had an old version (1.4) sitting in my downloads, which I had used once, while version 1.6 was installed in my applications.

When I right-clicked on the mf3 file and selected Open With, I noticed there were two ThumbHost3mf options: one from applications and the other from downloads (1.4).

After deleting the 1.4 version in downloads, Quick Look is now functioning as expected!
